src/main/java/uk/gov/companieshouse/web/lfp/controller/lfp/EnterLFPDetailsController.java

```diff
@@ -46,6 +47,9 @@ public class EnterLFPDetailsController extends BaseController {
     private static final String LFP_ONLINE_PAYMENT_UNAVAILABLE = "/online-payment-unavailable";
 
     private static final String PENALTY_TYPE = "penalty";
+    private static final String TEMPLATE_NAME_MODEL_ATTR = "templateName";
+    private static final String ENTER_LFP_DETAILS_MODEL_ATTR = "enterLFPDetails";
+    private static final String BACK_BUTTON_MODEL_ATTR = "backButton";
```

Comment:
This should be acquired from the baseController method addBackPageAttributeToModel